pkg/cli/login.go

```diff
@@ -79,7 +75,7 @@ func login(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	console.Infof("You've successfully authenticated as %s! You can now use the '%s' registry.", username, registryHost)
+	console.Infof("You've successfully authenticated as %s! You can now use the %s registry.", username, registryHost)
```

Comment:
This seems like an unrelated change
```suggestion
	console.Infof("You've successfully authenticated as %s! You can now use the '%s' registry.", username, registryHost)
	```

Reply:
yeah i thought about that but wanted to keep the changes minimal